adoc/MAIN.release-notes.adoc

```diff
@@ -114,54 +132,38 @@ providing improved flexibility and maintainabilitty to all {productname} users.
 
 We will strive to maintain {productname}'s compatibility with the Docker Engine in the future.
 
-=== Cilium Replaces Flannel
+==== Cilium Replaces Flannel
 
 {productname} now uses Cilium {cilium_version} as the Container Networking
 Interface enabling networking policy support.
 
 
-== Centralized logging
+=== Centralized logging
 
 The deployment of a Centralized Logging node is now supported for the purpose of
 aggregating logs from all the nodes in the kubernetes cluster.
 Centralized Logging forwards system and Kubernetes cluster logs to a
 specified external logging service, specifically the Rsyslog server,
 using Kubernetes Metadata Module - `mmkubernetes`.
 
-== Obsolete Components
+=== Obsolete Components
 
-=== Salt
+==== Salt
 
 Orchestration of the cluster no longer relies on Salt.
 Orchestration is instead achieved with `kubeadm` and `skuba`.
 
-=== Admin Node / Velum
+==== Admin Node / Velum
 
 The Admin Node is no longer necessary. The cluster will now be controlled
-by the master nodes and through API with `skuba` on the local workstation.
+by the master nodes and through API with `skuba` on any SUSE Linux enterprise system, such as a local workstation.
```

Comment:
The capitalization of SUSE Linux Enterprise is wrong. To avoid this mistake, just use the `{sle}` attribute.